src/js/dataMapping/covid19/recipient/map/map.js

```diff
@@ -101,42 +101,40 @@ export const filters = {
                 label: 'Individuals'
             }
         ]
-    },
-    awardType: {
-        label: 'AWARD TYPE',
-        options: [
-            {
-                value: 'all',
-                label: 'All Award Types'
-            },
-            {
-                value: 'contract',
-                label: 'Contracts'
-            },
-            {
-                value: 'idv',
-                label: 'Indefinite Delivery Vehicle'
-            },
-            {
-                value: 'grants',
-                label: 'Grants'
-            },
-            {
-                value: 'direct_payments',
-                label: 'Direct Payments'
-            },
-            {
-                value: 'loans',
-                label: 'Loans'
-            },
-            {
-                value: 'other',
-                label: 'Other'
-            }
-        ]
     }
 };
 
+export const awardTypeFilters = [
```

Comment:
This is fine for now, but we're re-defining this list in multiple sections of the page, and I'd eventually like to consolidate in [dataMapping/search/awardType.js](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/dataMapping/search/awardType.js). That will also make it easier when it comes time to break out insurance into its own tab across the site.